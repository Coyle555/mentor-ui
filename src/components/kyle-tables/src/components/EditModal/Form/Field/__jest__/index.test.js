import React from 'react';
import { Footer } from '../index';
import renderer from 'react-test-renderer';
import { cleanup, fireEvent, render, wait } from '@testing-library/react';
