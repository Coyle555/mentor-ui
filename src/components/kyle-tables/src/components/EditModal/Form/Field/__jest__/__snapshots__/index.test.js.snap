// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering a file field 1`] = `
<div>
  {"disabled":false,"type":"file","value":""}
</div>
`;

exports[`Rendering an color field 1`] = `
<div>
  {"disabled":false,"type":"color","value":""}
</div>
`;

exports[`Rendering an image field 1`] = `
<div>
  {"disabled":false,"type":"image","value":""}
</div>
`;

exports[`Rendering field with a parse 1`] = `
<div>
  {"disabled":false,"type":"string","value":"foo"}
</div>
`;

exports[`Rendering options that is a list filter 1`] = `
<div>
  {"disabled":false,"type":"listfilter","value":"","options":["foo"]}
</div>
`;

exports[`Rendering options that is not a list filter 1`] = `
<div>
  {"disabled":false,"type":"select","value":"","options":["foo"]}
</div>
`;

exports[`Rendering text field 1`] = `
<div>
  {"disabled":false,"type":"string","value":""}
</div>
`;
